From a2d53de5579dff95a582b92fa71f30cc28883546 Mon Sep 17 00:00:00 2001
From: Cody Xie <xkd@rock-chips.com>
Date: Fri, 15 Feb 2013 16:22:36 +0800
Subject: [PATCH] rk3188 ds1006h : do no invoke act8846_device_shutdown when
 shutdown without charger pluged in.                  this would cause tablet
 cannot boot while plug in charger                  and it can boot normally
 by press power button

---
 arch/arm/mach-rk3188/board-rk3188-ds1006h.c | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/arch/arm/mach-rk3188/board-rk3188-ds1006h.c b/arch/arm/mach-rk3188/board-rk3188-ds1006h.c
index c232bbc47462..38b6c7701a40 100755
--- a/arch/arm/mach-rk3188/board-rk3188-ds1006h.c
+++ b/arch/arm/mach-rk3188/board-rk3188-ds1006h.c
@@ -1806,10 +1806,13 @@ static void rk30_pm_power_off(void)
                        printk("enter restart===========\n");
                        arm_pm_restart(0, NULL);
                }
+		/** code here may cause tablet cannot boot when shutdown without charger pluged in
+		  * and then plug in charger. -- Cody Xie
                else
 		{
 			act8846_device_shutdown();
 		}
+		  */
        }
 #endif
 	gpio_direction_output(POWER_ON_PIN, GPIO_LOW);
-- 
2.35.3

